test_runner/src/main/kotlin/ftl/util/Utils.kt

```diff
@@ -155,6 +155,10 @@ fun withGlobalExceptionHandling(block: () -> Int) {
                 System.err.println(t)
                 exitProcess(1)
             }
+            is FlankNoSuchFileException -> {
+                System.err.println(t)
```

Comment:
I think user is only interested in message. We don't need to expose our classes. We could print just message here. (this is more cosmetically comment :) )

Other think that just came to my mind if we should stop using `System.err` to print messages. I am wondering if we should stick to stdout. Since it is CLI app user would by default redirect stderr somewhere else. Just thinking, what do you think?

Reply:
For change to stdout I will make other issue.